clmm/theory/parent_class.py

```diff
@@ -771,28 +1224,143 @@ def eval_magnification_bias(self, r_proj, z_cl, z_src, alpha):
             The projected radial positions in :math:`M\!pc`.
         z_cl : float
             Galaxy cluster redshift
-        z_src : array_like, float
-            Background source galaxy redshift(s)
+        z_src : array_like, float, function
+            Information on the background source galaxy redshift(s). Value required depends on
+            `z_src_info` (see below).
         alpha : float
             Slope of the cummulative number count of background sources at a given magnitude
+        z_src_info : str, optional
```

Comment:
Same docstring comments as before for the whole new redshift-related docstrings.